Userland/Libraries/LibC/pty.cpp

```diff
@@ -56,12 +56,22 @@ int openpty(int* amaster, int* aslave, char* name, const struct termios* termp,
         return -1;
     }
     if (termp) {
-        // FIXME: error handling
-        tcsetattr(*aslave, TCSAFLUSH, termp);
+        if (tcsetattr(*aslave, TCSAFLUSH, termp) == -1) {
+            int error = errno;
```

Comment:
That reads an uninitialized variable (`errno`) which just so happens to shadow the global `errno` variable.